docs/How-To/README.md

```diff
@@ -1,3 +1,3 @@
 # How-to guides
 
-The [How-To](./) folder contains how-to guides for different aspects of the Comsos project. Additional resources can be found in the [seed scripts](../../scripts/seeds) folder. 
+The [How-To](./) folder contains how-to guides for different aspects of the Cosmos project. Additional resources can be found in the [seed scripts](https://github.com/allinbits/cosmos-cash/tree/main/scripts/seeds) folder. 

```

Comment:
```suggestion
The [How-To](./) folder contains how-to guides for different aspects of the Cosmos project. Additional resources are in the [seed scripts](https://github.com/allinbits/cosmos-cash/tree/main/scripts/seeds) folder. 

```

Reply:
"Cosmos project" do we mean a blockchain that is built using the Cosmos SDK or do we mean generally "Cosmos ecosystem"?